tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -82,12 +82,19 @@ public SecuritySettings(IConfigurationSource? source, IConfigurationTelemetry te
                                  .AsDouble(val => val is <= 1 and >= 0)
                                  .GetValueOrDefault(0.1);
 
+            ApiSecurityMaxConcurrentRequests = config
+                                              .WithKeys(ConfigurationKeys.AppSec.ApiSecurityMaxConcurrentRequests)
+                                              .AsInt32(val => val >= 0)
```

Comment:
Is 0 a valid setting?

Reply:
changed to 1 in bca72f692faf563e74bd285e3d25c427ffc78f5d